app/controllers/content.js

```diff
@@ -56,7 +56,7 @@ export default Ember.Controller.extend(Analytics, {
     expandedAuthors: true,
     showLicenseText: false,
     fileDownloadURL: '',
-    expandedAbstract: false,
+    expandedAbstract: navigator.userAgent.includes('Prerender'),
```

Comment:
For the future, we may want to think about doing other things (like meta tags) conditionally on prerender as well, and centralizing the logic. But this is a good start.